mlflow/langchain/__init__.py

```diff
@@ -253,13 +257,6 @@ def load_retriever(persist_directory):
                 "file path containing the code for defining the chain instance."
             )
 
-        if len(code_paths) != 1:
-            raise mlflow.MlflowException.invalid_parameter_value(
-                "When the model is a string, there should be a config path provided. "
-                "This config path is used to set config.yml file path "
-                "for the model. This path should be passed in via the code_paths. "
-                f"Current code paths: {code_paths}"
-            )
```

Comment:
We should still throw if the length is greater than 1 :). Can we fix this @sunishsheth2009 ?

Reply:
Can we also add a test case verifying that we throw for length > 1?